src/components/AddPlaidBankAccount/plaidBankPropTypes.js

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /** Fired when the user exits the Plaid flow */
+    onExitPlaid: PropTypes.func,
+
+    /** Fired when the user selects an account and submits the form */
+    onSubmit: PropTypes.func,
+
+    /** Additional text to display */
```

Comment:
This comment isn't really helpful IMHO

Reply:
Ah yeah this happens with `propTypes` comments quite often because there's not much else to say about them and they are required. 

We're not taking full advantage of them yet, but if we use something like `react-docgen` we can generate component API docs from these comments - also they are used by Storybook (which we have set up, but also are not widely using yet).